crates/biome_js_analyze/src/lint/nursery/use_explicit_type.rs

```diff
@@ -461,6 +465,38 @@ fn is_function_used_in_argument_or_array(func: &AnyJsFunction) -> bool {
     )
 }
 
+/// Check if the function is used in some object
+///
+/// # Examples
+///
+/// JS_OBJECT:
+///
+/// interface Behavior {
+///   attribute: string;
+///   func: () => string;
+///   arrowFunc: () => string;
+/// }
+///
+/// function getObjectWithFunction(): Behavior {
+///   return {
+///     attribute: 'value',
+///     func: function myFunc(): string { return "value" },
+///     arrowFunc: () => {},
+///   }
+/// }
+///
+fn is_function_used_in_object(func: &AnyJsFunction) -> bool {
+    matches!(
+        func.syntax().parent().kind(),
+        Some(JsSyntaxKind::JS_PROPERTY_OBJECT_MEMBER)
+    )
+}
+
+/// Check if a function is an arrow function
+fn is_arrow_func(func: &AnyJsFunction) -> bool {
```

Comment:
@Conaclos 

Can you be more precise about this comment? https://github.com/biomejs/biome/pull/4233/files#r1821191207

Sorry, I don't have all the contextr

Reply:
TBH I don't remember. Looking at the code this looks fine.